docs/cheatsheet/cheatsheet.html

```diff
@@ -167,7 +167,7 @@
 
 <ul>
   <li>Assertions</li>
-  <li>Grey Assert
+  <li>UI Assertions
```

Comment:
Call this "Macros" instead?
GREYAsserts are not strictly for UI, we can use if for example to assert on timing in the test.